lib/punycode.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
 
+const { getOptionValue } = require('internal/options');
+if (getOptionValue('--pending-deprecation')){
```

Comment:
```suggestion
if (getOptionValue('--pending-deprecation')) {
```

Linter doesn't process this file as it's in `.eslintignore`.